### DISCUSSION TITLE
Capteur ant+ température

### DISCUSSION THREAD

**Discussion by Valery-04** [osmandapp/OsmAnd#18200] [General]
Bonjour,
Je viens d'installer Osmand et je trouve cette appli géniale !
J'ai réussi à connecter un capteur de température (Garmin) dans les capteurs externes et j'arrive à voir la température. En revanche je n'arrive pas à l'afficher dans les panneaux de configuration de l'écran. Je ne trouve pas le widget ? 
Merci de votre aide.
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/145754539/19b4e97c-bd84-46c2-8a2e-e718c1dfa11c"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
Please see https://osmand.net/docs/user/widgets/info-widgets#-external-sensors-widgets

**Comment by Valery-04** [comment]
Merci beaucoup de votre retour mais dans la configuration de l'écran / panneau de droite je n'ai pas le widget "température"
Merci de votre retour
![Uploading Screenshot_20230922_235458_OsmAnd.jpg…]()

**Reply by DmitryAlexei** [reply]
It will be available soon, in 4.6 release. You can join beta and try if it's working for you.
Also see https://github.com/osmandapp/OsmAnd/issues/17921
